pro-planner-client/src/index.js

```diff
@@ -49,6 +50,10 @@ const router = createBrowserRouter([
                         path: '',
                         element: <SchedulePage />
                     },
+                    {
+                        path: '',
```

Comment:
This route seems to clash with the schedule page. We may want to change either schedule page's path or this path. Might be fine to just change SchedulePage to `schedule` or something. This will require changing all instances that should lead to schedule page, which I believe may just be the link in navigation bar.